### PR TITLE
CE-3591 Pass title to VE on publish

### PR DESCRIPTION
**Pull request by kamilkoterba**
**Merge after https://github.com/Wikia/mercury/pull/2280**

## Links

* http://wikia-inc.atlassian.net/browse/CE-3591
* Related app PR https://github.com/Wikia/app/pull/10198

## Context

This PR is a continuation of work on CE-3552 as a extension of MVP.
See actual changes here done as a part of CE-3591 https://github.com/Wikia/mercury/compare/CE-3552...CE-3591-c and merge after CE-3552 PR mentioned no the top is merged.

## Description

Replaced isPublish boolean with title to use it on VE side.

## Reviewers

@Wikia/community-engineering @Wikia/west-wing @Wikia/x-wing

### CODE REVIEW

**Comment by Grunny** [comment]
Specific changes for this PR LGTM.

File: front/main/app/controllers/infobox-builder.js [hunk 1]
```diff
@@ -47,6 +52,16 @@ export default Ember.Controller.extend({
 			return this.get('model').getTemplateExists(title);
 		},
 
+		/**
+		 * Triggers action to communicate VE that creating infobox is completed
+		 *
+		 * @param {string} title Title of newly created infobox template
+		 * @returns {void}
+         */
```
Comment (bognix) [inline front/main/app/controllers/infobox-builder.js]: please fix indentation

Comment (Grunny) [inline front/main/app/controllers/infobox-builder.js]: @bognix That's part of and already fixed in the main PR (https://github.com/Wikia/mercury/pull/2280), this one is just the changes https://github.com/Wikia/mercury/compare/CE-3552...CE-3591-c :)

File: front/main/app/controllers/infobox-builder.js [hunk 2]
```diff
@@ -47,6 +52,16 @@ export default Ember.Controller.extend({
...
+		goBackToVE(title) {
```
Comment (bognix) [inline front/main/app/controllers/infobox-builder.js]: could you add some default value to title? Or check if title is passed or not - I see that sometimes you trigger this action with parameter and sometimes without

File: front/main/tests/unit/components/infobox-builder-test.js [hunk 3]
```diff
@@ -563,24 +607,29 @@ test('handleSaveResults', function (assert) {
...
 				message: 'naming conflict with redirect'
 			}
 		];
 
 	cases.forEach((testCase) => {
 		const redirectToPageSpy = sinon.spy(),
+			goBackToVESpy = sinon.spy(),
 			showEditTitleModalSpy = sinon.spy();
 
 		component.set('showEditTitleModal', showEditTitleModalSpy);
```
Comment (bognix) [inline front/main/tests/unit/components/infobox-builder-test.js]: you can use component.setProperties

File: front/main/app/controllers/infobox-builder.js (hunk 1 above)
Comment (bognix) [inline front/main/app/controllers/infobox-builder.js]: ack - thanks @Grunny

**Comment by bognix** [comment]
minor notes - LGTM

**Pull request merged**